crates/pg_workspace/src/workspace/server.rs

```diff
@@ -150,13 +157,28 @@ impl WorkspaceServer {
         Ok(())
     }
 
+    fn is_ignored_by_migration_config(&self, path: &Path) -> bool {
+        let set = self.settings();
+        set.as_ref()
+            .migrations
+            .as_ref()
+            .and_then(|migration_settings| {
+                let ignore_before = migration_settings.after.as_ref()?;
+                let migrations_dir = migration_settings.path.as_ref()?;
+                let migration = migration::get_migration(path, migrations_dir)?;
+
+                Some(&migration.timestamp <= ignore_before)
+            })
+            .unwrap_or(false)
```

Comment:
What happens if we pass a migrations_dir but no `after`? Wouldn't the `and_then` return `None`, so the function would always return `false`?

Reply:
yes, and we do not want to ignore any migration file if no `after` is specified do we?